packages/core/src/web/helpers/layer/convertBitmapToInfilledRect.ts

```diff
@@ -0,0 +1,50 @@
+import NS from '@core/app/constants/namespaces';
+
+/**
+ * convertBitmapToInfilledRect
+ * simplify bitmap to infilled rect when calculating contours
+ */
+const convertBitmapToInfilledRect = (): (() => void) => {
+  const svgcontent = document.getElementById('svgcontent');
+
+  if (!svgcontent) {
+    return () => {};
+  }
+
+  const images = svgcontent.querySelectorAll('image');
+  const oldElemMap: Array<{ elem: Element; nextSibling: Node | null; parentNode: Node | null }> = [];
+  const newElems: Element[] = [];
+
+  images.forEach((elem) => {
+    const rect = document.createElementNS(NS.SVG, 'rect');
+
+    ['x', 'y', 'width', 'height', 'transform'].forEach((attr) => {
+      const value = elem.getAttribute(attr);
+
+      if (value) {
+        rect.setAttribute(attr, value);
+      }
+    });
+    rect.setAttribute('fill', '#000');
+    rect.setAttribute('fill-opacity', '1');
+    elem.parentNode?.appendChild(rect);
```

Comment:
nitpick: `elem.parentNode?.insertBefore(rect, elem);` may be better if the order matter.
But the order does not matter in most case